docs/docs/widgets/overview.md

```diff
@@ -5,7 +5,7 @@ title: Overview
 
 # Components: Overview
 
-Components are used to build the UI of the applications. They can be dragged onto the canvas from the Component Library and can be modified from the Properties Panel without needing to write any code. **[Event Handlers](/docs/actions/event-handler)** in Components allow end users to trigger queries and other application events to perform the **[Actions](/docs/category/actions-reference)**.
+Components are used to build the UI of the applications. They can be dragged onto the canvas from the Component Library and can be modified from the Properties Panel without needing to write any code. **[Event Handlers]( https://docs.tooljet.com/docs/widgets/overview/#component-event-handlers)** in Components allow end users to trigger queries and other application events to perform the **[Actions](/docs/category/actions-reference)**.
```

Comment:
Hey @GODINME , Thanks for the PR.
It's suggested to follow the existing formatting in URLs,
Kinldy, replace the URL https://docs.tooljet.com/docs/widgets/overview/#component-event-handlers with `docs/widgets/overview#component-event-handlers`

Reply:
Okay @tejakummarikuntla

Reply:
@tejakummarikuntla  Just realized I made a space typo and have fixed it, Do you still want me to replace the link with your suggestion or push the verified correction made